.gitignore

```diff
@@ -1,2 +1,3 @@
 /_build
 /_exts
```

Comment:
``/_exts`` can be removed now.
It was here to avoid people submitting changes to the submodule reference by mistake, but we don't have anything here anymore